Documentation/user-guides/prometheus-agent.md

```diff
@@ -0,0 +1,175 @@
+---
+weight: 151
+toc: true
+title: Prometheus Agent
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: false
+description: Guide for running Prometheus in Agent mode
+---
+
+Since Prometheus-Operator `v0.64.0` it is also possible to run Prometheus in Agent mode.
+As mentioned in [Prometheus's blog](https://prometheus.io/blog/2021/11/16/agent/), Prometheus Agent
+is a deployment model optimized for environments where all collected data is forwarded to
+a long-term storage solution, e.g. Cortex or Thanos, and local Prometheus do not need storage or rule evaluation.
+
+To make it easier to upgrade from an older version to `v0.64.0` or higher, the operator is capable of running without
+the permissions to reconcile the PrometheusAgent CR. So to use the new CR, make sure to update the operator permissions to allow reconciliations of the new CR.
```

Comment:
```suggestion
the permissions to reconcile the PrometheusAgent CR. So to use the new PrometheusAgent CR make sure to update the operator permissions to allow reconciliations of the new CR.
```

Reply:
Ended up rephrasing it completely following Simon's suggestion, what do you think?